### PR TITLE
Feature/kotlin okhttp code gen

### LINKED ISSUE #243: Please add support for Kotlin khttp
**Is your feature request related to a problem? Please describe.**
Kotlin khttp is nice, commonly used library for requests, so it would be perfect that Postman has also possibility to export request code to khttp

### LINKED ISSUE #491: Add Support for Kotlin OkHttp code gen
**Is your feature request related to a problem? Please describe.**

Due the increased usage of Kotlin in the Android & JVM world, I would like to see more Kotlin code generators. As the language evolves rapidly over last years it should be part of Postman.

More than the half of all Android projects in the world already use Kotlin as their primary programming language:
[Kotlin ecosystem report 2020](https://www.jetbrains.com/lp/devecosystem-2020/kotlin/)

The superior network library OkHttp from Square is also rewritten in Kotlin:
[OkHttp 4.x](https://square.github.io/okhttp/upgrading_to_okhttp_4/)

**Describe the solution you'd like**

Similar to **java-okhttp** code generator only in Kotlin. 😄 

**Additional context**

We at [NanoGiants](https://github.com/nanogiants) use OkHttp for all of our Android projects, so I hope we can get this code gen in a future postman release.

### PR DESCRIPTION
**Pull request by nilsjr**
Here it is. Kotlin OkHttp code generator.

Resolves #491

### CODE REVIEW

**Review by jmayes41 — APPROVED** [review]

**Comment by nilsjr** [comment]
Hey is it possible to get a decision to merge it or not?

**Comment by dhwaneetbhatt** [comment]
Thank you for contributing @nilsjr. We will be reviewing this PR in the next 2 weeks, and suggest any improvements with the aim of merging.

File: codegens/kotlin-okhttp/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# codegen-kotlin-okhttp
+
+> Converts Postman-SDK Request into code snippet for kotlin-okhttp.
+
+#### Prerequisites
+To run Code-Gen, ensure that you have NodeJS >= v6. A copy of the NodeJS installable can be downloaded from https://nodejs.org/en/download/package-manager.
```
Comment (dhwaneetbhatt) [inline codegens/kotlin-okhttp/README.md:6]: We recently bumped the requirements for this module to NodeJS 12. We can update the docs here to convey this information.

File: codegens/kotlin-okhttp/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "type": "git",
+    "url": ""
+  },
+  "author": "Postman Labs <help@getpostman.com>",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/postmanlabs/code-generators/tree/master/codegens/kotlin-okhttp",
+  "dependencies": {},
+  "devDependencies": {},
+  "engines": {
+    "node": ">=8"
```
Comment (dhwaneetbhatt) [inline codegens/kotlin-okhttp/package.json:32]: As discussed, the minimum NodeJS version should be 12.

**Review by dhwaneetbhatt — CHANGES_REQUESTED** [review]
@nilsjr requested some minor changes. Also, could you please update your fork with the latest upstream changes from `develop`? We have some changes in CI (moved to GitHub actions) and would want a green CI check before merging the PR.

**Comment by nilsjr** [comment]
Yes will update the pull request in the next days. Almost 2 years old this one :D

**Comment by nilsjr** [comment]
@dhwaneetbhatt Updated and ready to review. Lets see if the workflow is running :D

File: codegens/kotlin-okhttp/package.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    "test-newman": "node npm/test-newman.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": ""
+  },
+  "author": "Postman Labs <help@getpostman.com>",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/postmanlabs/code-generators/tree/master/codegens/kotlin-okhttp",
+  "dependencies": {},
```
Comment (dhwaneetbhatt) [inline codegens/kotlin-okhttp/package.json:29]: We'd need to add `lodash` here as a dependency.

Comment (nilsjr) [reply codegens/kotlin-okhttp/package.json]: Added

**Review by dhwaneetbhatt — APPROVED** [review]

**Pull request merged**